decidim-core/app/packs/src/decidim/external_domain_warning.js

```diff
@@ -1,33 +1,26 @@
 /* eslint-disable require-jsdoc */
 
-export default function updateExternalDomainLinks($target) {
-  const whitelist = window.Decidim.config.get("external_domain_whitelist") || []
-
+export default function updateExternalDomainLinks(element) {
```

Comment:
Renamed to `ExternalDomainLink` back into #10121.
When #10967  will be merged, this will cause issues.

Reply:
Yes, we annotate this, when the sync is merged we'll take care of fixing.